apps/meteor/app/lib/server/functions/deleteUser.ts

```diff
@@ -15,19 +16,15 @@ import { LivechatUnitMonitors } from '../../../../ee/app/models/server';
 
 export async function deleteUser(userId: string, confirmRelinquish = false): Promise<void> {
 	const user = Users.findOneById(userId, {
-		fields: { username: 1, avatarOrigin: 1, federation: 1, roles: 1 },
+		fields: { username: 1, avatarOrigin: 1, roles: 1, federated: 1 },
 	});
 
 	if (!user) {
 		return;
 	}
 
-	if (user.federation) {
-		const existingSubscriptions = Subscriptions.find({ 'u._id': user._id }).count();
-
-		if (existingSubscriptions > 0) {
-			throw new Meteor.Error('FEDERATION_Error_user_is_federated_on_rooms');
-		}
+	if (isUserFederated(user)) {
+		throw new Meteor.Error('error-user-is-federated', 'Cannot delete federated user');
```

Comment:
I remember raising some concerns about this on the original PR but don't recall what came of it. Why is this even a requirement?

Reply:
Yes, I remember, we've agreed we should discuss that internally and find a solution to this, having in mind GDPR rules, but it was not discussed yet, would you suggest removing this validation instead?

Reply:
@pierre-lehnen-rc I'm reverting this change for now.